configure.ac

```diff
@@ -214,6 +214,14 @@ AC_C_BIGENDIAN(
 
 AC_SUBST([BUNDLER], ["$bundle_cmd"])
 
+case "$host" in
+    *mingw*)
+        LIBS="$LIBS -lshlwapi"
```

Comment:
In Makefile.am we have this:

```
if WIN32
libjq_la_LIBADD += -lshlwapi
endif
```

Ah, but we need that for jq too, not just libjq.  I think I'd rather change Makefile.am than configure.ac for this.